colorcet/__init__.py

```diff
@@ -7436,7 +7448,7 @@ def all_original_names(group=None, not_group=None, only_aliased=False, only_CET=
 
 
 
-linear_ternary_red_0_50_c52 = [  # cmap_def
+linear_ternary_red_0_50_c52: list[list[Union[int, float]]] = [  # cmap_def
```

Comment:
```suggestion
linear_ternary_red_0_50_c52: list[list[float]] = [  # cmap_def
```

I'm pretty sure `int` is a subset of `float` with type annotation.